meta-ostro/classes/ostro-image.bbclass

```diff
@@ -156,7 +280,21 @@ FEATURE_PACKAGES_soletta-tools = "soletta-dev-app"
 # because it is the most common source code management tool.
 FEATURE_PACKAGES_tools-develop = "packagegroup-core-buildessential git"
 
-FEATURE_PACKAGES_swupd = "packagegroup-swupd"
+# Add bash because it is more convenient to use than dash.
+# This does not change /bin/sh due to re-organized update-alternative
+# priorities.
+FEATURE_PACKAGES_tools-interactive = "packagegroup-tools-interactive bash"
```

Comment:
please add more useful helpers for interactive use, like bash-completion? :-)

I'm thinking the case where we have all daemons split in command line clients and actual server, in that case should we add the cmdline clients here? like connmanctl, systemctl, systemd console agents... It would be nice to make this more automatic, so cmdline tools split packages can be auto pulled in.

nothing to be done now, but let's think about it.